web-src/src/components/MainSidePanel.js

```diff
@@ -190,12 +201,18 @@ export function MainSidePanel(props) {
               && <Text>{formatMessage(intlMessages.mainSidePanel.recentsMenuItem)}</Text>}
             </li>
             {(mainSidePanelType === MainSidePanelType.Expanded && sessions && sessions.length > 0)
-              && sessions.map((session) => (
-                // eslint-disable-next-line max-len
-                <li className={currentSession && viewType === ViewType.CurrentSession && session && session.id === currentSession.id ? derivedStyles.clickedSubMenuItem : style.subMenuItem} key={session.id}>
-                  <Link href="#" UNSAFE_className={style.menuItemLink} onPress={() => handleRecent(session)}>{session.name}</Link>
-                </li>
-              ))}
+              && sessions.toReversed().map((session) => {
+                const sessionDate = new Date(session.name);
+                const groupingLabel = groupingLabelGenerator(sessionDate);
+
+                return (<>
```

Comment:
this will lead to an warning  in the dev console:
` index.js:1 Warning: Each child in a list should have a unique "key" prop.`

To prevent it, you need to add key on the element inside loop.  See https://medium.com/@ishaqibrahimbss/reacts-favorite-warning-each-child-in-a-list-should-have-a-unique-key-prop-909dfe50cbe3

I'll fix it in the scope of https://github.com/adobe/aem-genai-assistant/pull/273, no action needed for now